swagger.yaml

```diff
@@ -183,8 +185,12 @@ paths:
           type: string
         - name: source_metadata
           in: query
-          description: '???'
+          description: >-
+            A JSON blob containing extra source specific information.  This
+            field is free-form, but we generally expect the data will follow
+            one of the formats outlined in the `/schema` folder.
```

Comment:
Will that interfere with Swagger? The server may return HTML rather than a YAML file when swagger queries it.  If Swagger uses GitHub it may instead get an erb file.